### ISSUE TITLE
Update from 20.0 to 27.0 forces node pool recreation

### ISSUE THREAD

**Issue by justsomecorporateuser** [terraform-google-modules/terraform-google-kubernetes-engine#1773]
### TL;DR

Seems that random_id keepers list forces recreation of node pools

### Expected behavior

While nothing other changed than module version, would expect that existing node pools remain.

### Observed behavior

Terraform plan shows that random_id is recreated and new values are added under keepers. This causes 

      ~ name                        = "pool-xxxx-9492" -> (known after apply) # forces replacement
...

  # module.gke.random_id.name["pool-xxxx-8"] must be replaced
+/- resource "random_id" "name" {
      ~ b64_std     = "pool-xxxx-8-lJI=" -> (known after apply)
      ~ b64_url     = "pool-xxxx-8-lJI" -> (known after apply)
      ~ dec         = "pool-xxxx-38034" -> (known after apply)
      ~ hex         = "pool-xxxx-9492" -> (known after apply)
      ~ id          = "lJI" -> (known after apply)
      ~ keepers     = { # forces replacement
          + "enable_gcfs"                 = ""
          + "enable_gvnic"                = ""
          + "enable_integrity_monitoring" = ""
          + "enable_secure_boot"          = ""
          + "max_pods_per_node"           = ""
          + "min_cpu_platform"            = ""
          + "pod_range"                   = ""
          + "spot"                        = ""
            # (13 unchanged elements hidden)
        }
        # (2 unchanged attributes hidden)
    }

### Terraform Configuration

```hcl
beta-private-cluster-update-variant, no other changes than version from 20.0 to 27.0
```

### Terraform Version

```sh
0.14.6
```

**Comment by SergeySevruk** [comment]
The same issue with 28.00

**Comment by mike-serchenia** [comment]
The same issue with 29.00
